<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditConfigWindow</class>
 <widget class="QDialog" name="EditConfigWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>194</width>
    <height>88</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit config</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="2">
    <widget class="ConfigSettingsWidget" name="configSettingsWidget" native="true"/>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="editPushButton">
       <property name="text">
        <string>Edit</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-ok"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelPushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-cancel"/>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="baseInfoLabel">
     <property name="text">
      <string>Editing config </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConfigSettingsWidget</class>
   <extends>QWidget</extends>
   <header>widgets.ConfigSettingsWidget.ConfigSettingsWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>EditConfigWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
